src/Mssql/ExpressionTrait.php

```diff
@@ -23,7 +23,7 @@ protected function _escape(string $value): string
 
     // {{{ MSSQL does not support named parameters, so convert them to numerical inside execute
 
-    private $paramsBackup;
+    private $paramsBackup = [];
```

Comment:
it seems my fix is not working, but this is WRONG - we either need to use a backup or not